docs/website/docs/general-usage/naming-convention.md

```diff
@@ -66,16 +58,20 @@ In the snippet below, we define a resource with various "illegal" unicode charac
 ### Understand the identifier normalization
 Identifiers are translated from source to destination form in **normalize** step. Here's how `dlt` picks the right naming convention:
 
-* Each destination has a preferred naming convention.
-* This naming convention is used when new schemas are created.
-* Schemas preserve naming convention when saved
-* `dlt` applies final naming convention in `normalize` step. Naming convention comes from (1) explicit configuration (2) from destination capabilities. Naming convention
-in schema will be ignored.
-* You can change the naming convention in the capabilities: (name, case-folding, case sensitivity)
+* Each destination may define a preferred naming convention (ie. Weaviate), otherwise **snake case** will be used
+* This naming convention is used when new schemas are created. This happens when pipeline is run for a first time.
+* Schemas preserve naming convention when saved. Your running pipelines will maintain existing naming conventions if not requested otherwise
+* `dlt` applies final naming convention in `normalize` step. Naming convention comes from (1) explicit configuration (2) from destination capabilities.
+* Naming convention will be used to put destination is case sensitive/insensitive mode and apply the right case folding function.
+
+:::caution
```

Comment:
Maybe we should add here that we do not recommend to change the naming convention for an existing dataset and if need to do it, a fresh load to a new dataset would be the least errorprone.

Reply:
this documentation needs at least a day of work...